backend/e4h-services/ingestion-service/app/api/endpoints/template_generation.py

```
@@ -37,6 +38,63 @@
+    request_info = request_info_from_json(payload.get("request_info", {}))
+    boundary_data = payload.get("boundary_data", {})
+    get_authorized_request_info(request_info)
+    mdms_client = MDMSClient(mdms_url)
+    try:
+        timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+        output_filename = f"facility_ingestion_template_{timestamp}.xlsx"
+        output_file_path = create_temp_file(suffix=".xlsx")
+        try:
+            facility_schema = mdms_client.get_column_definitions_with_metadata(request_info, 'data-ingestion.FacilityIngestionSchema')
+            boundary_list: List[Boundary] = flatten_boundaries(boundary_data)
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)